### PR TITLE
Use the region of the first Airnode deployment for the bucket location

### LINKED ISSUE #1487: Use the region of the first Airnode deployment for the bucket location
[Slack discussion](https://api3workspace.slack.com/archives/C02AYRX8D89/p1664799843233839)

### PR DESCRIPTION
**Pull request by amarthadan**
Close #1487

### CODE REVIEW

File: packages/airnode-deployer/src/infrastructure/aws.ts [hunk 1]
```diff
@@ -31,15 +32,48 @@ export const getAirnodeBucket = async () => {
...
+export const createAirnodeBucket = async (cloudProvider: AwsCloudProvider) => {
   const s3 = initializeS3Service();
   const bucketName = generateBucketName();
 
-  logger.debug(`Creating S3 bucket '${bucketName}'`);
-  const goCreate = await go(() => s3.createBucket({ Bucket: bucketName }).promise());
+  let createParams: AWS.S3.CreateBucketRequest = { Bucket: bucketName };
+  // If the region is `us-east-1` the configuration must be empty...
+  // https://docs.aws.amazon.com/AWSJavaScriptSDK/latest/AWS/S3.html#createBucket-property
+  if (cloudProvider.region !== 'us-east-1') {
```
Comment (Siegrift) [inline packages/airnode-deployer/src/infrastructure/aws.ts:71]: 🙈

File: packages/airnode-deployer/src/infrastructure/aws.ts [hunk 2]
```diff
@@ -31,15 +32,48 @@ export const getAirnodeBucket = async () => {
...
+
+  let region = goBucketLocation.data.LocationConstraint;
+  // The `EU` option is listed as a possible one in the documentation
+  // https://docs.aws.amazon.com/AWSJavaScriptSDK/latest/AWS/S3.html#getBucketLocation-property
+  if (isNil(region) || region === 'EU') {
+    throw new Error(`Unknown bucket region '${region}'`);
+  }
+  // The documentation says that for buckets in the `us-east-1` region the value of `LocationConstraint` is null but it is actually an empty string...
+  // https://docs.aws.amazon.com/AWSJavaScriptSDK/latest/AWS/S3.html#getBucketLocation-property
+  if (region === '') {
```
Comment (Siegrift) [inline packages/airnode-deployer/src/infrastructure/aws.ts:54]: 🙈

Comment (amarthadan) [reply packages/airnode-deployer/src/infrastructure/aws.ts:54]: :man_shrugging: I'm not happy about it either. Take it up with the AWS SDK developers :smile:

**Review by Siegrift — APPROVED** [review]
👍 LGTM

**Pull request merged**